modules/react/skeleton/lib/skeleton.tsx

```diff
@@ -4,11 +4,11 @@ import styled from '@emotion/styled';
 
 import {accessibleHide, createComponent, StyledType} from '@workday/canvas-kit-react/common';
 
-import SkeletonHeader from './parts/skeletonHeader';
-import SkeletonShape from './parts/skeletonShape';
-import SkeletonText from './parts/skeletonText';
+import {SkeletonHeader} from './parts/skeletonHeader';
+import {SkeletonShape} from './parts/skeletonShape';
+import {SkeletonText} from './parts/skeletonText';
```

Comment:
Initially, `ArgsTable` wasn't working for Skeleton subcomponents (e.g., `<ArgsTable of={Skeleton.Text} />` wasn't displaying anything). After consulting with @NicholasBoll, it turns out we needed to convert the default exports in the Skeleton subcomponents to named exports to get their `ArgsTables` to populate properly. These subcomponents aren't exported by the module so this isn't a breaking change.